themes/custom/droopler_theme/scss/components/_d_p_text_with_bckg.scss

```diff
@@ -22,13 +22,30 @@ $d-p-text-with-bckg-img-opacity: 0.1 !default;
 
 // Override default values.
 .padding-top-default {
-  .paragraph--type--d-p-text-with-bckg {
+  .d-p-text-with-bckg {
     padding-top: 0;
+
+    .content-wrapper {
+      padding-top: $d-p-text-with-bckg-padding-top;
+
+      @include media-breakpoint-down(sm) {
+        padding-top: $d-p-text-with-bckg-mobile-padding-top;
+      }
+    }
   }
 }
+
 .padding-bottom-default {
-  .paragraph--type--d-p-text-with-bckg {
+  .d-p-text-with-bckg {
     padding-bottom: 0;
+
+    .content-wrapper {
+      padding-bottom: $d-p-text-with-bckg-padding-bottom;
+
+      @include media-breakpoint-down(sm) {
+        padding-bottom: $d-p-text-with-bckg-mobile-padding-bottom;
+      }
+    }
   }
 }
 
```

Comment:
![image](https://user-images.githubusercontent.com/37533838/87312825-5e9a8900-c521-11ea-9f9c-3068e58e3373.png)

o to mi chodziło, zeby wlasnie wszedzie uzywac tych klas `paragraph--type--d-p-text-with-bckg` a nie skrócone wersje, które pojawiają się w tagach głębiej

Reply:
to co jest teraz nie zadzaiała w oczekiwany sposob:
![image](https://user-images.githubusercontent.com/37533838/87313264-ebdddd80-c521-11ea-903d-12b0f81e91a4.png)